browser-app.cpp

```diff
@@ -353,8 +357,13 @@ bool BrowserApp::Execute(const CefString &name, CefRefPtr<CefV8Value>,
 			 const CefV8ValueList &arguments,
 			 CefRefPtr<CefV8Value> &, CefString &)
 {
-	if (name == "getCurrentScene" || name == "getStatus" ||
-	    name == "saveReplayBuffer") {
+	if (name == "getControlLevel" || name == "getCurrentScene" ||
+	    name == "getStatus" || name == "startRecording" ||
+	    name == "stopRecording" || name == "startStreaming" ||
+	    name == "stopStreaming" || name == "pauseRecording" ||
+	    name == "unpauseRecording" || name == "startReplayBuffer" ||
+	    name == "stopReplayBuffer" || name == "startVirtualcam" ||
+	    name == "stopVirtualcam" || name == "saveReplayBuffer") {
```

Comment:
I did some simplistic timing and it seems that an `IsValidFunction(name)` function that uses a vector or an array with std::find generally performs as good or better than this approach.  This approach only performs better if the sought function name is in one of the first few conditions checked (i.e., up to startRecording).  The savings are minute, but it looks nicer.

If you don't want to change this in this PR, I don't consider it a blocker, and I can address it later.

Reply:
Done